components/Cards/docs/accessibility.md

```diff
@@ -0,0 +1,66 @@
+### Accessibility Labels
+
+To help ensure your Cards are accessible to as many users as possible, please be sure to review the following 
+recommendations:
+
+#### Accessibility Labels for Nested Elements
+
+Set an appropriate[`accessibility
+Label`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619577-accessibilitylabel) 
+for meaningful content that is not textual by default, like images. Also make sure to enable the element's 
+[`isAccessible
+Element`](https://developer.apple.com/documentation/objectivec/nsobject/1615141-isaccessibilityelement) 
+property so it is detectable by assistive technologies.
+
+Swift
+```swift
+myImageView.isAccessibilityElement = true
```

Comment:
Nested elements are completely user driven, so I don't see how MDCCard can disable it. I assume images are not accessible elements by default since there is no sensible default text that can be used for them.